viewing_party/party.py

```diff
@@ -1,23 +1,225 @@
 # ------------- WAVE 1 --------------------
+import pprint
 
 def create_movie(title, genre, rating):
-    pass
+    """Create dictionary containing title, genre, and rating of a movie
+    
+    Keyword arguments:
+    title -- the title of the movie
+    genre -- the genre of the movie
+    rating -- the rating of the movie
+    """
+    # Return None if any parameter is falsy
+    if not title or not genre or not rating:
+        return None
+        
+    movie = {
+        "title": title, 
+        "genre": genre,
+        "rating": rating
+    }
+
+    return movie
+
+def add_to_watched(user_data, movie):
+    """Add a movie to watched list.
+    
+    Keyword arguments:
+    user_data -- dictionary containing movies user has and wants to watch
+    movie -- dictionary containing title, genre, and rating
+    """
+    # Check movie isn't empty and is a dictionary
+    # Try to add movie to watched, otherwise give error message
+    if movie and isinstance(movie, dict):
+        try:
+            user_data["watched"].append(movie)
+        except KeyError:
+            print("'Watched' key does not exist; could not add movie")
+    
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    """Add a movie to a watchlist.
+    
+    Keyword arguments:
+    user_data -- dictionary containing movies user has and wants to watch
+    movie -- dictionary containing title, genre, and rating
+    """
+    # Check movie isn't empty and is a dictionary
+    # Try to add movie to watchlist, otherwise give error message
+    if movie and isinstance(movie, dict):
+        try:
+            user_data["watchlist"].append(movie)

```

Comment:
[suggestion]

consider constants for the dictionary keys

When working with dictionary keys (which can be typo prone) it's sometimes helpful to make constants that our editor is better able to suggest for us. If we had

```python
KEY_WATCHLIST = "watchlist"
```

defined elsewhere, we could write

```python
    watch_list = user_data[KEY_WATCHLIST]
```

We might get better suggestions from our editor, reducing typos. If in the future a key ever needed to change, we'd also only need to change it in one place. Refactoring shortcuts can also help, in a good editor, but sometimes we can help things along ourselves!